keps/sig-cli/0031-kustomize-integration.md

```diff
@@ -32,122 +33,210 @@ superseded-by:
 * [Table of Contents](#table-of-contents)
 * [Summary](#summary)
 * [Motivation](#motivation)
-    * [Goals](#goals)
-    * [Non-Goals](#non-goals)
-* [Kustomize Introduction](#kustomize-introduction)    
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Kustomize Introduction](#kustomize-introduction)   
 * [Proposal](#proposal)
-    * [UX](#UX)
-      * [apply](#apply)
-      * [get](#get)
-      * [delete](#delete)
-    * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
-    * [Risks and Mitigations](#risks-and-mitigations)
+  * [UX](#UX)
+     * [apply](#apply)
+     * [get](#get)
+     * [delete](#delete)
+  * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+  * [Risks and Mitigations](#risks-and-mitigations)
 * [Graduation Criteria](#graduation-criteria)
 * [Implementation History](#implementation-history)
 * [Alternatives](#alternatives)
 
 [Tools for generating]: https://github.com/ekalinin/github-markdown-toc
 
 ## Summary
-[Kustomize](https://github.com/kubernetes-sigs/kustomize) was developed as a distinct tool from kubectl to rapidly experiment on declarative configuration. The code was developed to solve some long standing, complex kubectl [issues](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/0008-kustomize.md#long-standing-issues). The final step is to close the issues by bringing the code into kubectl. This KEP describes how to do this with consistent kubectl UX.
+
+[Kustomize](https://github.com/kubernetes-sigs/kustomize)
+was developed as a subproject of sig-cli by kubectl maintainers with the goal of addressing
+a collection of issues creating friction for declarative workflows in kubectl (e.g. `kubectl apply`).  The
+goal of Kustomize was to eventually bring this functionality back to kubectl in order to complement
+`kubectl apply` and other declarative workflow commands.
+
+- declaratively generating Resource config
+- declaratively transforming Resource config
+- composing collections of Resource config across files and directories
+- layering the above on top of one another
+
+It is independent of, but complementary to, the *server-side apply* initiative that was started later and targeted
```

Comment:
Link to https://github.com/kubernetes/enhancements/issues/555 for lazy people like myself :smile: